recipes/abqpy/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "abqpy" %}
+{% set version = "2025.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: fd9ba620cfe44a94e68fbf60ab90db3bc7febdc5cf7e11658fa60c21c58b6fc5
+
+build:
```

Comment:
This package looks like it should probably be `noarch: python`. See:
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python

If that's the case, please revise the recipe as show in the linked docs.